go/ethclient/erc20contractlib/obscuro_ERC20_contract_ABI.go

```diff
@@ -0,0 +1,62 @@
+package erc20contractlib
```

Comment:
Nit: use consts for hardcoded value

Reply:
which value?

Reply:
There are a few hardcoded values in this file, like the `4` or the `transfer`.
I'm fine with leaving them as is when they are only used once, but I think the standard we're following was no hardcoded values ?